### PR TITLE
Docs/compodoc

### PR DESCRIPTION
**Pull request by jnizet**
on top of #22

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,16 +32,18 @@
...
+    "@angular/cli": "~6.0.7",
+    "@angular/compiler-cli": "^6.0.3",
     "@angular/language-service": "^6.0.3",
+    "@compodoc/compodoc": "^1.1.3",
     "@types/jasmine": "2.8.8",
     "@types/jasminewd2": "2.0.3",
     "@types/node": "8.9.5",
     "codecov": "3.0.2",
     "codelyzer": "4.2.1",
+    "event-stream": "^3.3.4",
```
Comment (cexbrayat) [inline package.json]: Is this required?

**Comment by cexbrayat** [comment]
Reviewing the generated doc, I was thinking: shouldn't we rename all components and service with the `Valdemort` prefix for coherence?

File: package.json (hunk 1 above)
Comment (jnizet) [reply package.json]: Bizarrely, yes. I think compdoc upgraded just before added compodoc, and forgot a peer dependency, making the build fail. I'll retry with the latest version to see if it's fixed.

**Comment by jnizet** [comment]
shouldn't we rename all components and service

> The component and directives aren't really part of the public API. Only their selector is, in fact. So it doesn't matter much. And I don't really like long names starting with a meaningless prefix.

**Comment by cexbrayat** [comment]
Yeah that's why I only renamed the others, but I was thinking about Augury for example, which will display the components on screen. But that's OK

**Comment by jnizet** [comment]
Aaaarg. NPM is striking again. It randomly adds or removes packages in my node_modules. Running npm run doc on my Mac works with the latest version of the files I pushed, but it fails on Travis. I'll try pushing a commit which uses yarn, to see if it males a difference.

**Comment by jnizet** [comment]
Aaaand... it works fine with yarn. I'll make the switch. I'm sick of fighting with NPM.

**Pull request merged**